bioimageio/spec/model/v0_4/schema.py

```diff
@@ -166,9 +166,24 @@ def matching_halo_length(self, data, **kwargs):
             raise NotImplementedError(type(shape))
 
 
-class PytorchStateDictWeightsEntry(_WeightsEntryBase):
+class _WeightsEntryBase(_WeightsEntryBase03):
     raw_nodes = raw_nodes
+    dependencies = fields.Dependencies(
+        bioimageio_description="Dependency manager and dependency file, specified as `<dependency manager>:<relative "
+        "path to file>`. For example: 'conda:./environment.yaml', 'maven:./pom.xml', or 'pip:./requirements.txt'. "
+        "These dependencies are only valid in the context of the specified weight format."
```

Comment:
```suggestion
        "These dependencies are only used for the specified weight format."
```

Reply:
I think "used for" is shorter and more accurate here. But feel free to stick with your original formulation if you prefer it.